srcjs/shinyapp.js

```diff
@@ -663,7 +663,11 @@ var ShinyApp = function() {
       // render the HTML and deps to a null target, so
       // the side-effect of rendering the deps, singletons,
       // and <head> still occur
-      exports.renderHtml($([]), message.content.html, message.content.deps);
+      try {
+        exports.renderHtml($([]), message.content.html, message.content.deps);
+      } catch(err) {
+        throw "The selector you chose could not be found in the DOM.";
```

Comment:
Yeah it looks like this call has transposed arguments, should be:

`exports.renderHtml(message.content.html, $([]), message.content.deps)`